### PR TITLE
Makes sleepers buildable and adds them to the research techweb

### LINKED ISSUE #6: Constructible Sleepers with dropping machine boards.

### PR DESCRIPTION
**Pull request by FernandoJ8**
## About The Pull Request

Sleepers can now be built and deconstructed like a normal machine, as well as having their own research node that makes them printable at a medical lathe.

## Why It's Good For The Game

Closes #6, sleepers being available in-round is cool and fits a more combat-focused style.

## Changelog

:cl:
add: Sleepers can now be built and deconstructed. A research node has been also added that makes the boards printable at a medical lathe.
/:cl:

### CODE REVIEW

**Comment by FernandoJ8** [comment]
We're not actually doing changelogs, are we?

**Comment by san7890** [comment]
> We're not actually doing changelogs, are we?

i think we should tbh

File: code/modules/research/designs/machine_designs.dm [hunk 1]
```diff
@@ -516,6 +516,14 @@
 	category = list(RND_CATEGORY_MEDICAL_MACHINERY)
 	departmental_flags = DEPARTMENT_BITFLAG_MEDICAL
 
+/datum/design/board/sleeper
+	name = "Machine Design (Sleeper Unit Board)"
+	desc = "The circuit board for a sleeper unit"
```
Comment (san7890) [inline code/modules/research/designs/machine_designs.dm:521]:
```suggestion
	desc = "The circuit board for a sleeper unit."
```

**Review by san7890 — CHANGES_REQUESTED** [review]

**Review by JohnFulpWillard — COMMENTED** [review]
deconstructing a stasis unit needs to drop the board

**Comment by JohnFulpWillard** [comment]
> > We're not actually doing changelogs, are we?
> 
> i think we should tbh

it will get overwriten when we update to latest TG

**Comment by FernandoJ8** [comment]
> deconstructing a stasis unit needs to drop the board


> deconstructing a stasis unit needs to drop the board

The sleeper board? Why?

**Pull request merged**